app/models/subscription_fixed_charge_units_override.rb

```diff
@@ -11,7 +11,7 @@ class SubscriptionFixedChargeUnitsOverride < ApplicationRecord
   belongs_to :subscription
   belongs_to :fixed_charge
 
-  validates :units, presence: true, numericality: {greater_than_or_equal_to: 0}
+  validates :units, presence: true, numericality: {greater_than: 0}
```

Comment:
why is zero not an option now.

Allowing units to be zero it was covering a customer use case where the fixed charge is defined without units, then changed to a value (for example in the first invoice) with the goal for example to charge in the first subscription invoice a one off fee.

Reply:
ah, when I was writing tests, I didn't think about this case and exactly was questioning why would we create a fixed charge with 0 units🙈 
thank you, updating 🙌